src/sagemaker/tensorflow/estimator.py

```diff
@@ -229,7 +229,7 @@ def __init__(self, training_steps=None, evaluation_steps=None, checkpoint_path=N
             **kwargs: Additional kwargs passed to the Framework constructor.
         """
         if framework_version is None:
-            LOGGER.warning(fw.empty_framework_version_warning(TF_VERSION, TF_VERSION))
+            LOGGER.warning(fw.empty_framework_version_warning(TF_VERSION, self.LATEST_VERSION))
         self.framework_version = framework_version or TF_VERSION
```

Comment:
Still you haven't changed TF_VERSION to LATEST_VERSION

Reply:
The initial reported issue is that TF_VERSION is set '1.11' and LATEST_VERSION is set '1.12'. TF_VERSION should be replaced by LATEST_VERSION. Or TF_VERSION should be updated to 1.12

Reply:
I understand that the transition from having an updating default value to requiring `framework_version` be explicitly defined could be better, and I apologize for the inconvenience. There's more explanation in #754 around this, and there'll be a separate PR for the documentation changes proposed in #754.